### PR TITLE
header on large screen

### PR DESCRIPTION
**Pull request by aninarafath6**

### CODE REVIEW

File: components/Home/Header/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Image from "next/image";
+import { Images } from "../../../constants/constants";
```
Comment (shihabus) [inline components/Home/Header/Logo.jsx]: try using `constants/constants`.  Of not working, try adding `jsconfig.js`

File: components/Home/Header/SideBar.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (shihabus) [inline components/Home/Header/SideBar.jsx]: Do you need to explicitly import React? I guess it is available by default

File: components/Home/Header/actions.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect, useRef } from "react";
+import Lottie from "react-lottie";
```
Comment (shihabus) [inline components/Home/Header/actions.jsx:3]: [Good to have] to group lib imports and local imports, good to leave a new line

File: components/Home/Header/actions.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import animationData from "../../../public/icons/Menu V3/menuV3.json";
+import SideBar from "./SideBar";
```
Comment (shihabus) [inline components/Home/Header/actions.jsx:8]: [Good to have] add a new line

File: components/Home/Hero/HeroSection.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Image from "next/image";
+
+export default function HeroSection() {
+  return (
+    <div className="flex px-6 justify-evenly py-9 flex-col md:flex-row items-center hero-section">
+      <div className="md:items-start  items-center  flex flex-col justify-center text-center md:text-start ">
+        <h1 className="font-semibold md:text-6xl lg:text-7xl text-5xl md:text-start  text-center	 ">
+          A perfect <span className="inline-block  md:hidden">road</span>map
```
Comment (shihabus) [inline components/Home/Hero/HeroSection.jsx:11]: [Good to have]Can we create a constant file for all the texts? In that way, it is much easier to edit and update.

File: constants/constants.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import { Images } from "./image-constants";
+
+export { Images };
```
Comment (shihabus) [inline constants/constants.js]: we can do this as
`export * from "./image-constants"`

Comment (shihabus) [reply constants/constants.js]: rename this file as `index.js`. So during import, it can be just `constants`

File: pages/_app.jsx [hunk 7]
```diff
@@ -1,7 +1,13 @@
 import "../styles/globals.css";
```
Comment (shihabus) [inline pages/_app.jsx]: [Good to have] keep local imports below lib/package imports

**Review by shihabus — CHANGES_REQUESTED** [review]

File: components/Home/Hero/HeroSection.jsx (hunk 5 above)
Comment (aninarafath6) [reply components/Home/Hero/HeroSection.jsx:11]: that's cool

File: components/Home/Header/LoginButton.jsx [hunk 8]
```diff
@@ -0,0 +1,14 @@
+export default function LoginButton({ size = "sm" }) {
+  console.log(size);
```
Comment (shihabus) [inline components/Home/Header/LoginButton.jsx]: remove the log later

File: components/Home/Header/Logo.jsx [hunk 9]
```diff
@@ -1,5 +1,5 @@
 import Image from "next/image";
-import { Images } from "../../../constants/constants";
+import { Images } from "constants/index";
```
Comment (shihabus) [inline components/Home/Header/Logo.jsx:2]: no need of `index` here

File: components/Home/Hero/HeroSection.jsx [hunk 10]
```diff
@@ -1,28 +1,35 @@
-import React from "react";
 import Image from "next/image";
+import { Images, Contents } from "constants/index";
```
Comment (shihabus) [inline components/Home/Hero/HeroSection.jsx:2]: +1

File: components/Home/Hero/SearchInput.jsx [hunk 11]
```diff
@@ -0,0 +1,26 @@
+import { Images } from "constants/index";
```
Comment (shihabus) [inline components/Home/Hero/SearchInput.jsx:1]: +1

**Review by shihabus — APPROVED** [review]
good if we can clean up the requested changes. Even without that it is OK to deploy, no worries.

File: components/Home/Header/Logo.jsx (hunk 9 above)
Comment (aninarafath6) [reply components/Home/Header/Logo.jsx:2]: I tried it but not working , so I chosen this method

File: components/Home/Header/LoginButton.jsx (hunk 8 above)
Comment (aninarafath6) [reply components/Home/Header/LoginButton.jsx]: done✅

**Review by shihabus — APPROVED** [review]

**Pull request merged**